src/Makefile.am

```diff
@@ -121,19 +119,23 @@ souffle_sources = \
 	      InterpreterProgInterface.h                \
 	      InterpreterPreamble.h			\
 	      InterpreterRecords.h         InterpreterRecords.cpp     \
+              RamAnalysis.h                             \
               RamComplexityAnalysis.cpp  RamComplexityAnalysis.h  \
-              RamLevelAnalysis.cpp  RamLevelAnalysis.h  \
               RamCondition.h                            \
+              RamExpression.h                           \
+              RamIndexAnalysis.cpp  RamIndexAnalysis.h  \
               RamNode.h                                 \
+              RamLevelAnalysis.cpp  RamLevelAnalysis.h  \
               RamOperation.h                            \
               RamProgram.h                              \
               RamRelation.h                             \
               RamStatement.h                            \
               RamTransformer.cpp    RamTransformer.h    \
               RamTransforms.cpp     RamTransforms.h     \
               RamTranslationUnit.h                      \
-              RamExpression.h                           \
+              RamTypes.h                                \
               RamVisitor.h                              \
+              RamUtils.h                                \
```

Comment:
If you are ordering lexicographically here, U comes before V.

You could also consider ordering the rest of the list, and converting the few tabs to spaces so the entries line up consistently.

Reply:
Perhaps we defer this to another pull-request.